front/package.json

```diff
@@ -106,6 +106,7 @@
     "react-dates": "21.8.0",
     "react-dnd": "^16.0.1",
     "react-dnd-cjs": "^9.5.1",
+    "react-dnd-html5-backend": "^16.0.1",
```

Comment:
I don't see this one imported anywhere, should we keep it? Or maybe we can get rid of the old `cjs` versions instead?

Reply:
It's used in `PostManager`.

Reply:
Eventually the cjs versions will go.